src/scripts/printInterface.js

```diff
@@ -0,0 +1,44 @@
+import { getElement, createElement } from './queries.js';
+
+function subMenu() {
```

Comment:
- [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.